source/Calamari/Kubernetes/Conventions/Helm/HelmManifestAndStatusReporter.cs

```diff
@@ -108,9 +120,15 @@ async Task<string> PollForManifest(RunningDeployment deployment,
                     await Task.Delay(TimeSpan.FromSeconds(1), ct.Token);
                 }
 
-            if (string.IsNullOrWhiteSpace(manifest))
+            //it's possible that some manifests doesn't
+            if (string.IsNullOrWhiteSpace(manifest) && !didSuccessfullyExecuteCliCall)
                 throw new CommandException("Failed to retrieve helm manifest in a timely manner");
 
+            //Log if we found a manifest, or not
+            log.Verbose(string.IsNullOrWhiteSpace(manifest)
+                            ? $"Retrieve an empty manifest for {releaseName}, revision {revisionNumber}."
```

Comment:
```suggestion
                            ? $"Retrieved an empty manifest for {releaseName}, revision {revisionNumber}."
```
Will also need to update the test assertions